### PR TITLE
25425

### PR DESCRIPTION
**Pull request by nt-nclarity**
Recreated ClientList project with VS React Template and CRA.  Added react-router-dom and bootstrap dependencies.  Created basic routing.

### CODE REVIEW

File: ClientList/ClientList/.gitignore [hunk 1]
```diff
@@ -0,0 +1,232 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (marcdeotis) [inline ClientList/ClientList/.gitignore]: Nice! 💯

File: ClientList/ClientList/ClientApp/src/Pages/Clients/ClientsPage.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class ClientsPage extends Component {
+    render(){
+        return(
+            <div>
```
Comment (marcdeotis) [inline ClientList/ClientList/ClientApp/src/Pages/Clients/ClientsPage.js]: I like the clean styling of this page!

File: ClientList/ClientList/ClientApp/src/App.js [hunk 3]
```diff
@@ -1,28 +1,25 @@
...
-        </header>
-      </div>
-    );
-  }
+    render() {
+        return (
+            <Router>
+                <div>
+                    <ul>
+                        <li><Link to="/subscriptions">Subscriptions</Link></li>
```
Comment (marcdeotis) [inline ClientList/ClientList/ClientApp/src/App.js]: Should we add a tooltip to make sure people know what these links are for?  😄

File: ClientList/ClientList/Startup.cs [hunk 4]
```diff
@@ -1,20 +1,32 @@
...
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.SpaServices.ReactDevelopmentServer;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 
-namespace ClientList
+namespace ClientList2
```
Comment (marcdeotis) [inline ClientList/ClientList/Startup.cs]: Shouldn't our namespace match the project name?

File: ClientList/ClientList/ClientApp/src/App.js (hunk 3 above)
Comment (nt-nclarity) [reply ClientList/ClientList/ClientApp/src/App.js]: MVP2

File: ClientList/ClientList/Startup.cs (hunk 4 above)
Comment (nt-nclarity) [reply ClientList/ClientList/Startup.cs]: Yes, I'll fix it

**Pull request merged**